doc/proposals/TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 ## <Title of Proposal>
 
 Implementation Owner: < github username >
-Status: Draft
+Status: **`<Status>`** E.g draft/complete 
```

Comment:
I think this should be whatever we want the Status to be when the proposal author begins writing. 
Maybe:

```suggestion
<!--
  Possible Status values:
    - Draft
    - Accepted
    - In progress
    - Complete
    - Deprecated
    - Removed
-->
Status: Draft
```

Do we ever merge proposals before they are fully fleshed out? If not, we could change this from `Draft` to `Accepted` and remove the `Draft` option from the comment.

/cc @hasbro17 @estroz

Reply:
done 👍 tks for the input.

Reply:
I agree `Draft` should be removed.